watertap/core/util/property_helpers.py

```diff
@@ -0,0 +1,50 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pandas as pd
+
+
+def print_property_metadata(prop_pkg, return_df=False):
+    """
+    Print all supported properties from a WaterTAP/IDAES property package.
+
+    Args:
+        prop_pkg: The property model ParameterBlock (e.g., m.fs.properties)
+        return_df: If True, returns a Pandas DataFrame instead of printing.
+    """
+    metadata = prop_pkg.get_metadata()
+    vars, units, docs = [], [], []
+
+    for v in metadata.properties:
+        vars.append(v._name)
+        units.append(str(v._units))
+        docs.append(v._doc)
+
+    if return_df:
+        return pd.DataFrame(
+            {"Property Description": docs, "Model Attribute": vars, "Units": units}
```

Comment:
I would suggest column headers with no spaces. "Name", "Description", "Units". The prefix "Property" is implied in all cases.

Reply:
ok will do!

Reply:
Does it need to be a dataframe? Why not just a dict?